app/views/parks/index.html.erb

```diff
@@ -1,9 +1,15 @@
-<h1> Dog Parks in Massachusetts </h1>
+<h1> Dog Parks </h1>
 
 <ul>
   <% @parks.each do |park| %>
-    <li>
-      <%= park.name %>
-    </li>
+  <li>
+    <div class="callout">
+    <div class="callout-header"><%= park.name %></div>
```

Comment:
Please update your feature test for your index to include the heading, the address, and the link!